.travis.yml

```diff
@@ -8,7 +8,8 @@ addons:
 
 cache:
     directories:
-        - $HOME/.composer/cache/files
+        - $HOME/.cache/pip
+        - $HOME/.composer/cache
```

Comment:
Not actual inconsistencies, but rather delays in updated repository information. I got the intel from @xabbuh originally, I don't have any references. The relevand PR: https://github.com/php-http/boilerplate/pull/19